src/Configuration/ConfigurationFactory.php

```diff
@@ -189,6 +186,28 @@ private function retrievePhpUnit(SchemaConfiguration $schema, string $configDir)
         return $phpUnit;
     }
 
+    /**
+     * @return array<string, Mutator>
+     */
+    private function retrieveMutators(array $schemaMutators, string $mutatorsInput): array
+    {
+        if (count($schemaMutators) === 0) {
+            $schemaMutators = ['@default' => true];
+        }
+
+        $parsedMutatorsInput = $this->mutatorParser->parse($mutatorsInput);
+
+        if ([] === $parsedMutatorsInput) {
```

Comment:
what do you think about `[] === $parsedMutatorsInput` as we don't use yoda style here? Line 194 above have another style ;)

Reply:
I'll look into the CS-Fixer... I hate to have to think about that one